pkg/bisect/bisect.go

```diff
@@ -21,6 +21,7 @@ import (
 type Config struct {
 	Trace          debugtracer.DebugTracer
 	Fix            bool
+	Compiler       string
```

Comment:
To be honest, `Compiler` and `BisectCompiler` look very confusing together. Maybe rename `Compiler` to sth like `DefaultBisectCompiler` and add some comments?

Reply:
This is all in the bisect package and everything here is already Bisect-something. Adding Bisect to names does not add anything useful.
BisectCompiler is more like CompilerType b/c it contains a small set of predefined strings. While the Compiler can be DefaultCompiler.

Reply:
I agree with this, though this raises the question whether we also want to [change the ManagerConfig](https://github.com/google/syzkaller/blob/master/syz-ci/syz-ci.go#L158-L160). I think it's fine to change the BisectCompiler->CompilerType, because we are not yet using BisectCompiler in prod. However changing Compiler-> DefaultCompiler would require a deprecation period.

Reply:
How about we just go with CompilerType and Compiler?

Reply:
The `Compiler` in the `ManagerConfig` is not just used for bisection, so I think it's fine to leave it as is. But inside the `pkg/bisect` code we can surely do both changes.

Reply:
Implemented as suggested!